gateway/response/response.go

```diff
@@ -95,3 +99,25 @@ func GetStatusCode(key string) int {
 
 	return 200
 }
+
+//Always returns a valid response json
+func GetResponse(key string) string {
+	if status, ok := statusMap[key]; ok {
+		response, err := json.Marshal(status)
```

Comment:
`json.Marshal/unMarshal` is a generally a costly operation, it does mem allocs and runs some reflects.
I did a quick check and fmt.Sprintf seems to be taking half the time as json marshal.

Even though the time taken is small, since this happens for every event, how about we do this instead?
```
fmt.Sprintf(`{"msg": "%s"}`, response.Message)
```

Reply:
done